content/devops/foundational-short-course/2-why-is-devops-important/_index.md

```diff
@@ -0,0 +1,38 @@
+
+It entails a long planning phase, with strict time schedule, budget, and implementation of the entire project in one shot. Also, it brings strict documentation, a very formal review process, and sign off from IT ops between every phase.
+
+In one word, it is **inflexible**. Project’s demands changes, team changes, budget changes, features changes, management changes. Everything is very dynamic (and unstable!) throughout the software development process and DevOps came as a different way to think about it.
+
+DevOps became a thing in the late 2000s as a response for the inflexibility when developing software. The term was coined by Patrick Debois in 2009, when he organized the first DevOpsDays conference in Ghent, Belgium. Here the term became a buzzword. 
+
+There are DevOpsDays conferences all around the world now! You can check them here: https://devopsdays.org/.
+
+Another main event in the DevOps history is the release of The Phoenix Project, an IT novel, introducing DevOps concepts to the readers. It is amazing, you should add it to your reading list!
+
+Know more:
+- [Software development life cycle (SDLC) methodologies](https://en.wikipedia.org/wiki/Software_development_process#Methodologies)
+- [DevOpsDays Conferences](https://devopsdays.org/)
+- [The Phoenix Project](https://itrevolution.com/product/the-phoenix-project/)
+- [What is DevOps?](https://aws.amazon.com/devops/what-is-devops/?nc1=h_ls)
+- [The history of DevOps: a visual timeline](https://www.techtarget.com/whatis/reference/The-history-of-DevOps-A-visual-timeline)
+
+## What are the DevOps benefits?
+**Faster software delivery**: business can respond better to market changes due to software development automation and streamline. 
+
+**Better software quality**: using integration and automation tools, software errors and bugs can be detected and fixed earlier. 
+
+**Improved collaboration and efficiency**: high level of collaboration between different teams inside the organization, enabling a culture of knowledge sharing and best practices.
+
+## What are the DevOps downsides?
+**Increased complexity**: DevOps practices usually require the adoption and integration of multiple new tools and technologies, creating a more complex and heterogeneous software environment. Also, giving more autonomy to different teams might lead to a lack of standardization across the company. Having a standardized and battle-tested “DevOps toolchain” helps to mitigate these issues.
+
+**Cultural shock**: shifting an organization’s culture is a hard and slow process, which requires buy-in from the top management, trickling down to all the teams and employees. Creating a DevOps and breaking down the silos mindset will require a lot of training, communication and iteration between different groups of people.
```

Comment:
Organisation* 
Please check this content for British English spelling. I may miss stuff